python/GafferUI/VectorDataPlugValueWidget.py

```diff
@@ -81,16 +79,29 @@ def setHighlighted( self, highlighted ) :
 		GafferUI.PlugValueWidget.setHighlighted( self, highlighted )
 		self.vectorDataWidget().setHighlighted( highlighted )
 
-	def _updateFromPlug( self ) :
+	@staticmethod
+	def _valuesForUpdate( plugs ) :
+
+		assert( len( plugs ) == 1 )
+		plug = next( iter( plugs ) )
```

Comment:
Am I right in thinking that this is a synonym for `plugs[0]`?  It looks counterintuitive to me.

Reply:
If `plugs` was a `list` then yes, but it's a `set`. I believe this is the standard Python idiom for getting the first element.